Meetings/2023-01-17.md

```diff
@@ -0,0 +1,49 @@
+# PlanetaryPy TC Meeting January 17 2023 @ 12:30 PM PST / 1:30 PM MST / 2:30 PM CST / 3:30 PM EST / 20:30 UTC
+
+* Zoom Link: https://cuboulder.zoom.us/j/2789149686
+
+## Participants
+
+* Michael Aye (MA)
+* Andrew Annex (AA)
+
+## pending AIs
+
+## Agenda
+
+### replies to poll for new meeting time
+
+* Only Ross replied so far:
+  * http://whenisgood.net/nqan4gk
+  * Andrew added his availabilities now.
+
+### finish craterstats affiliation application
+* what's next step? (just clarification)
+  * We actually don't have a PR yet, so Michael will ask Greg to formally apply for the affiliation now.
```

Comment:
This isn't true.  Greg submitted one in March 2021: https://github.com/planetarypy/planetarypy.github.io/pull/5

Reply:
ah, we were searching for it, forgot that it's on another repo...